x-pack/plugins/security_solution/public/common/components/ml_popover/ml_popover.tsx

```diff
@@ -27,6 +27,8 @@ import { useSecurityJobs } from './hooks/use_security_jobs';
 
 const PopoverContentsDiv = styled.div`
   max-width: 684px;
+  max-height: 800px;
+  overflow: scroll;
```

Comment:
```suggestion
  max-height: 90vh;
  overflow-y: auto;
  overflow-x: hidden;
  padding-bottom: 15px;
```


Here's the magic we worked on over zoom 🙂